packages/mobile/src/account/__snapshots__/EditProfile.test.tsx.snap

```diff
@@ -50,20 +50,14 @@ exports[`renders the EditProfile Component 1`] = `
         placeholder="yourName"
         rejectResponderTermination={true}
         style={
-          Array [
```

Comment:
This PR has a bunch of  see snapshot updates exactly like this. I just noticed that our text input component had an unnecessary style array, which was merged together into one style. Feel free to ignore these